code/services/ExampleInsertAllRows/ExampleInsertAllRows.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Inserts all the rows in the given table
+ * @param {Object} req     
+ *      requestBody:{
```

Comment:
this is invalid JSDoc, you need to use @example

Reply:
My intention was something else, but got you.